tracker/data/processing.py

```diff
@@ -36,6 +36,45 @@
 SCAN_CACHE = os.path.join(env.SCAN_DATA, "cache")
 SCAN_DOMAINS_CSV = os.path.join(SCAN_CACHE, "domains.csv")
 
+ACCEPTED_CIPHERS = {
```

Comment:
@poteaduncan @buckley-w-david It'd be super nice if these were hosted by CSE or TBS **NOT** in a PDF form, such that an update to the cipher list would be able to easily allow this service to auto-update it's scan list without having to push a code-update.

Still don't like this in code, would rather this be an externally managed file. (still merge it though)

Reply:
I agree - it should be kept in a separate file similar to how the domain list is kept separate.